index.html

```diff
@@ -45,115 +45,133 @@
               <a class="navigation__link" href="#">Contact Us</a>
             </li>
           </ul>
-        </div>
+        </nav>
 
-        <button>Donate Now</button>
+        <button class = "donate_now">Donate Now</button>
 
-    </div>
+      </header>
 
-    <div class="content">
-        <div class="main">
+    <main class="content">
+        <main class="main">
           <div class="alert">
             You've helped us give <span class="text-highlight">72</span> bikes
             to refugees and asylum seekers so far. <strong>Thank you!</strong>
           </div>
 
           <div class="hero">
-            <h1>Bikes for Refugees</h1>
+            <h1>Bikes for Refugees </h1>
             <p>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class = "donate">Donate a bike today</button>
```

Comment:
*nitpick* same here, specifically, do `donate` and `donate_now` need their own separate classes?